src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -291,7 +291,7 @@ def version(self) -> int:
             self._version = self._get_next_version()
         return self._version
 
-    def _get_next_version(self):
+    def _get_next_version(self):  # type: ignore[no-untyped-def]
```

Comment:
Can we actually resolve types in here?

Reply:
I added an ignore here because I was asked to remove an annotation of self in a previous PR.

Should all `selfs` be a forward ref of the class, or marked as an ignore?

Reply:
Alright, seems I have missed that. That makes sense then